services/olvid.yml

```diff
@@ -0,0 +1,6 @@
+id: olvid
+name: Olvid
+rules:
+  - '||olvid.io^'
+  - '||olvid-attachment-chunks.s3.eu-west-3.amazonaws.com^'
+icon_svg: <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 50 50"><path d="M510 2475 c-184 -40 -350 -173 -433 -346 -74 -153 -72 -132 -72 -889 l0 -675 23 -80 c34 -117 83 -198 176 -290 59 -59 101 -90 156 -117 142 -70 129 -69 913 -66 695 3 703 3 760 25 173 66 303 177 374 320 75 149 74 139 71 923 -3 688 -3 696 -25 760 -33 96 -90 186 -163 260 -76 75 -148 120 -252 158 l-73 27 -695 2 c-567 1 -707 -1 -760 -12z m950 -399 c396 -95 660 -424 660 -821 0 -326 -171 -601 -465 -746 -149 -74 -230 -92 -405 -92 -173 0 -255 18 -397 88 -222 110 -383 305 -449 545 -25 91 -25 319 0 410 93 337 348 566 704 630 69 13 277 5 352 -14z"/><path d="M1135 1716 c-129 -32 -234 -118 -290 -239 -101 -214 -42 -477 138 -614 17 -14 25 -27 21 -37 -9 -23 -91 -81 -162 -114 l-62 -29 38 -7 c104 -17 314 2 442 40 175 52 328 174 395 315 50 105 64 209 43 316 -29 146 -100 254 -209 320 -99 59 -237 78 -354 49z" /></g></svg>
```

Comment:
Hello, @olivier3m. Thanks for your suggestion.
This SVG is invalid, please use the svg below instead:
```
<svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 250 250"><path d="M51 2.5c-18.4 4-35 17.3-43.3 34.6C.3 52.4.5 50.3.5 126v67.5l2.3 8c3.4 11.7 8.3 19.8 17.6 29 5.9 5.9 10.1 9 15.6 11.7 14.2 7 12.9 6.9 91.3 6.6 69.5-.3 70.3-.3 76-2.5 17.3-6.6 30.3-17.7 37.4-32 7.5-14.9 7.4-13.9 7.1-92.3-.3-68.8-.3-69.6-2.5-76-3.3-9.6-9-18.6-16.3-26-7.6-7.5-14.8-12-25.2-15.8l-7.3-2.7-69.5-.2c-56.7-.1-70.7.1-76 1.2zm95 39.9c39.6 9.5 66 42.4 66 82.1 0 32.6-17.1 60.1-46.5 74.6-14.9 7.4-23 9.2-40.5 9.2-17.3 0-25.5-1.8-39.7-8.8A85.66 85.66 0 0 1 40.4 145c-2.5-9.1-2.5-31.9 0-41 9.3-33.7 34.8-56.6 70.4-63 6.9-1.3 27.7-.5 35.2 1.4z"/><path d="M113.5 78.4a43.05 43.05 0 0 0-29 23.9c-10.1 21.4-4.2 47.7 13.8 61.4 1.7 1.4 2.5 2.7 2.1 3.7-.9 2.3-9.1 8.1-16.2 11.4l-6.2 2.9 3.8.7c10.4 1.7 31.4-.2 44.2-4 17.5-5.2 32.8-17.4 39.5-31.5 5-10.5 6.4-20.9 4.3-31.6-2.9-14.6-10-25.4-20.9-32-9.9-5.9-23.7-7.8-35.4-4.9z"/></svg>
```